src/api_authentication_app_skeleton/src/actions/api/password_reset_requests/create.cr

```diff
@@ -0,0 +1,14 @@
+class Api::PasswordResetRequests::Create < ApiAction
+  include Auth::RequireNotSignedIn
+
+  route do
+    PasswordResetRequestForm.new(params).submit do |form, user|
+      if user
+        PasswordResetRequestEmail.new(user).deliver
+        json({message: "You should receive an email on how to reset your password shortly"}, Status::OK)
```

Comment:
I believe you can do `:ok` instead of `Status::OK`. I'm not sure though. Also if we use the enums here I think they should be used everywhere so it isn't mixing and matching enums with raw int status codes

Reply:
What do you think about this? I think all status codes should use symbols if possible